devtools/bom-descriptor-json/src/main/resources/catalog-overrides.json

```diff
@@ -131,7 +131,7 @@
         "maven-plugin-artifactId": "quarkus-maven-plugin",
         "maven-plugin-version": "${project.version}",
         "gradle-plugin-id": "io.quarkus",
-        "gradle-plugin-version": "${project.version}",
+        "gradle-plugin-version": "${quarkus-gradle-plugin.version}",
```

Comment:
Why a different property? `maven-plugin-version` above uses the project.version too and I don't remember seeing any problem with it

Reply:
I added one for the Maven plugin as well for consistency. Depending on the Quarkus (platform) build, Maven and Gradle plugins may be different.